admiral/pkg/clusters/registry_test.go

```diff
@@ -2,16 +2,12 @@ package clusters
 
```

Comment:
Seems like you deleted these tests but moved the methods they are testing - let's make sure they are re-added in the right places